src/main/java/uk/gov/ons/census/fwmtadapter/model/dto/CollectionCase.java

```diff
@@ -33,4 +33,5 @@
   private Integer ceActualResponses;
   private Boolean addressInvalid;
   private Boolean undeliveredAsAddressed;
+  private boolean handDelivery;
```

Comment:
We probably got this wrong in Case Processor. Would prefer `Boolean` to `boolean`